src/main/java/eu/dissco/backend/domain/DefaultMappingTerms.java

```diff
@@ -57,7 +58,7 @@ private static Set<MappingTerm> fillAggregationList() {
     aggregationTerms.add(HAS_MEDIA);
     aggregationTerms.add(ORGANISATION_NAME);
     aggregationTerms.add(ORGANISATION_ID);
-    aggregationTerms.add(SOURCE_SYSTEM_ID);
+    aggregationTerms.add(SOURCE_SYSTEM_NAME);
```

Comment:
We could keep source system id as an aggregation term?

Reply:
We could but it doesn't say a lot to the user which is why we wanted to change it. You can still filter on it though so search?sourceSystemID=XXX/XXX would still work. But the list of aggregations would otherwise get a bit long (and eventuelly slow)